androidApp/src/main/java/com/mbta/tid/mbta_app/android/util/FormattedAlert.kt

```diff
@@ -30,6 +32,10 @@ data class FormattedAlert(
         @Composable
         get() = stringResource(R.string.effect_ahead, stringResource(downstreamEffectRes))
 
+    // TODO: Decide if should be nullable or not
+    val cause
```

Comment:
I think naming this `cause` might be confusing later if we want to move the title-case cause text into `FormattedAlert` at some point - the way this PR is set up right now, `@StringRes val dueToCauseRes: Int?` and `val delaysDueToCause @Composable get()` might be the structure that rhymes best with the `effect`/`downstreamEffect` distinction that's in here now.